rsqueakvm/constants.py

```diff
@@ -127,42 +127,42 @@
 SO_JIT_HOOK_RCVR = 59 # really selectorTrap
 
 constant_objects_in_special_object_table = {
-    "nil": (SO_NIL, "POINTERS"),
```

Comment:
We use maps for W_FixedPointersObjects that have inline fields. These objects are prebuilt constants - we need to create them to be the right kind of empty PBCs.